rakelib/docs.rake

```diff
@@ -9,9 +8,9 @@ namespace "docs" do
 
   task "generate-docs" do
     require "bootstrap/environment"
-
-    list = Dir.glob("#{LogStash::Environment.logstash_gem_home}/gems/logstash-*/lib/logstash/{input,output,filter,codec}s/*.rb").join(" ")
-    cmd = "bin/logstash docgen -o asciidoc_generated #{list}"
+    pattern = "#{LogStash::Environment.logstash_gem_home}/gems/logstash-*/lib/logstash/{input,output,filter,codec}s/*.rb"
+    list    = Dir.glob(pattern).join(" ")
```

Comment:
Nope, I use to have all ```=``` alligned. actually the way it was before it's uncommon as a coding style.